package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "test:e2e": "$(npm bin)/cypress run"
```

Comment:
Suggest to replace with `"./node_modules/.bin/cypress run"` as it works for both Windows and Mac

Reply:
Thanks! i've updated the script & tested locally. It seems fine on mac but i think if it works for Windows it should be no problem also.